static/app/utils/replays/replayDataUtils.tsx

```diff
@@ -113,6 +115,15 @@ export function breadcrumbFactory(
       url: initialUrl,
     },
   } as BreadcrumbTypeDefault;
+  const finalBreadcrumb = {
+    type: BreadcrumbType.INIT,
+    timestamp: replayRecord.finishedAt.toISOString(),
+    level: BreadcrumbLevelType.INFO,
+    data: {
+      action: 'replay-finish',
+      label: t('Finish recording'),
+    },
+  } as BreadcrumbTypeDefault;
```

Comment:
This is a new thing, before we only had the Start crumb.

<img width="410" alt="Screen Shot 2022-10-19 at 1 15 41 PM" src="https://user-images.githubusercontent.com/187460/196794825-712f2cee-985e-4bfb-a26b-21dbf9164c82.png">

I'm still verifying that this crumb will appear after any errors (especially if captureOnlyOnError is enabled)

Reply:
Start recording is only shown when we don't have an initial page load detected IIRC -- maybe we remove that condition and always show "Start Recording"